### DISCUSSION TITLE
Consider renaming Harp stream file naming conventions

### DISCUSSION THREAD

**Discussion by glopesdev** [SainsburyWellcomeCentre/aeon_experiments#262] [General]
With the new Harp auto-generated interfaces for output expanders and other Harp devices, it would now be possible to log all Harp timestamps using the actual register name, rather than the more abstract register address number.

This would look something like the following:

| Current | Proposal |
| :------- | :--------  |
| Synchronizer_8_2023-06-12T11-00-00.bin | Synchronizer_Heartbeat_2023-06-12T11-00-00.bin |
| Patch1_35_2023-06-12T11-00-00.bin | Patch1_OutputSet_2023-06-12T11-00-00.bin |

In any case we also need to consider whether / how to leverage the new device schemas to auto-generate readers and metadata for devices in Aeon experiments, with the goal of having the complete description of all logged streams available upon data acquisition.

**Comment by glopesdev** [comment]
@jkbhagatio @bruno-f-cruz @aspaNeuro Let's use this thread for discussion.

**Reply by bruno-f-cruz** [reply]
I think it is important to show everyone how you go from a device, logging data, and, finally, to automatically load data in python. It will help if everyone understands the current perks/limitations. I would propose doing this during the weekly Data Architecture meeting if it is not too late.

Regardless, I like the proposal, I am just wondering if the geal is to still split message types (namely reads) or to just put them together. This is because, since right now Events and Writes are saved in the same file, the decision to keep everything together has already been made in a way.
